dwave_networkx/drawing/pegasus_layout.py

```diff
@@ -129,50 +133,71 @@ def pegasus_node_placer_2d(G, scale=1., center=None, dim=2, crosses=False):
     """
     import numpy as np
 
-    m = G.graph.get('rows')
-    h_offsets = G.graph.get("horizontal_offsets")
-    v_offsets = G.graph.get("vertical_offsets")
-    tile_width = G.graph.get("tile")
+    line_plot = False if normalize_kwargs is None else normalize_kwargs.get('line_plot')
+
+    if line_plot:
+        crosses = False
+
+    m = G.graph['rows']
+    h_offsets = G.graph["horizontal_offsets"]
+    v_offsets = G.graph["vertical_offsets"]
+    tile_width = G.graph["tile"]
+    odd_k_wobble = .05
     tile_center = tile_width / 2 - .5
+    cross_shift = 2 if crosses else 0
 
     # want the enter plot to fill in [0, 1] when scale=1
-    scale /= m * tile_width
+    fabric_scale = m*tile_width - 2*odd_k_wobble - 1
+    scale /= fabric_scale
+
+    if normalize_kwargs is not None:
+        normalize_size_and_aspect(fabric_scale, 150, normalize_kwargs)
```

Comment:
where does 150 come from?

Reply:
It's a tight upper bound on the entries of lines 181 and 183; can you recommend clearer language for line 149?

Reply:
It's a magic value, determined experimentally.

Reply:
Is it worth defining it as a constant above line 153 with a comment and then using that here?

Reply:
I don't think so